Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.3.7"
 
 [deps]
 GeometryBasics = "5c1252a2-5f33-56bf-86c9-59e7332b4326"
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
```

Comment:
I'm guessing, that this was unintentional?

Reply:
No, I used `subtypes` in `precompile.jl`.

Reply:
Oh :D fair enough ;)